src/libs/ReceiptUtils.ts

```diff
@@ -66,7 +66,7 @@ function getThumbnailAndImageURIs(transaction: Transaction, receiptPath: string
         image = ReceiptSVG;
     }
 
-    const isLocalFile = typeof path === 'number' || path.startsWith('blob:') || path.startsWith('file:');
+    const isLocalFile = typeof path === 'number' || path.startsWith('blob:') || path.startsWith('file:') || path.startsWith('/');
```

Comment:
This PR created a problem with eReceipts that have pending waypoints. Issue https://github.com/Expensify/App/issues/34287